caom2pipe/manage_composable.py

```diff
@@ -1887,6 +1897,8 @@ def ftp_get(ftp_host_name, source_fqn, dest_fqn):
 
     Uses ftputil, which always transfers files in binary mode.
     """
+    # remove the need to have ftputil libraries on EVERY *2caom2 pipeline
+    from ftputil import FTPHost
```

Comment:
I moved it from the beginning of the file, because it requires an additional library install, and it's currently only NEOSSAT that requires the capability for ftp transfers.

I could move the functions to the neossat2caom2 code-base, if you think that would be better?